src/crawlers/info.ts

```diff
@@ -292,30 +292,71 @@ export const _hyundaihmallcom = () => {
 };
 
 export const _ssensecom = () => {
+  const brands = {
+    'A.P.C.': '아페쎄',
+    'Alexander McQueen': '알렉산더 맥퀸',
+    'Acne Studios': '아크네 스튜디오',
+    'ADER error': '아더에러',
+    'adidas Originals': '아디다스 오리지널스',
+    Balenciaga: '발렌시아가',
+    Balmain: '발망',
+    Burberry: '버버리',
+    'Calvin Klein Underwear': '캘빈클라인 언더웨어',
+    'Carhartt Work In Progress': '칼하트WIP',
+    'Coach 1941': '코치',
+    Converse: '컨버스',
+    'Dolce & Gabbana': '돌체앤가바나',
+    'Dr. Martens': '닥터마틴',
+    Fendi: '펜디',
+    Givenchy: '지방시',
+    Gucci: '구찌',
+    Kenzo: '겐조',
+    Ksubi: '수비',
+    Lacoste: '라코스테',
+    "Levi's": '리바이스',
+    'Maison Kitsuné': '메종 키츠네',
+    'Maison Margiela': '메종 마르지엘라',
+    MCQ: '멕큐',
+    'New Balance': '뉴발란스',
+    Nike: '나이키',
+    'Paul Smith': '폴스미스',
+    'Raf Simons': '라프 시몬스',
+    'Reebok Classics': '리복 클래식',
+    'Saint Laurent': '입생로랑',
+    Valentino: '발렌티노',
+    Vans: '반스',
+    Versace: '베르사체',
+  };
+
   const name = document
     .querySelector('meta[name="twitter:title"]')
     .getAttribute('content')
     .split('-')[1]
     .trim();
-  const brandKor = document
+  const brandEng = document
     .querySelector('meta[name="twitter:data2"]')
     .getAttribute('content');
+  const brandKor = brands[brandEng];
   const imageUrl = document
-    .querySelector('meta[property="og:image"]')
-    .getAttribute('content');
+    .querySelector('img.product-detail')
+    .getAttribute('data-srcset');
   const salePrice = document
     .querySelector('meta[name="twitter:data1"]')
     .getAttribute('content')
     .split(' ')[0]
     .slice(1);
```

Comment:
```suggestion
  const salePrice = document
    .querySelector('meta[name="twitter:data1"]')
    .getAttribute('content')
    .replace(/[^\d]+/g, '')
```

`$78 USD`을 `78`로 가공하는 로직을 여러가지 방법으로 구현할 수 있을텐데용
숫자를 제외한 나머지 문자를 모두 제거하는 방식으로 변경해봤습니다~